app/views/EditAry/AssessmentRegistryForm/AdditionalDocument/Preview/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useCallback, useMemo } from 'react';
+import { isDefined, isNotDefined } from '@togglecorp/fujs';
+import { Modal } from '@the-deep/deep-ui';
+
+import LeadPreview from '#components/lead/LeadPreview';
+import { useRequest } from '#base/utils/restRequest';
+
+import { FileUploadResponse } from '../types';
+
+interface Props {
+    attachmentId?: string;
+    onChangeSelectedAttachment: React.Dispatch<React.SetStateAction<string | undefined>>;
+    link?: string;
+}
+
+function Preview(props: Props) {
+    const {
+        attachmentId,
+        onChangeSelectedAttachment,
+        link,
+    } = props;
+
+    const handleModalClose = useCallback(() => {
+        onChangeSelectedAttachment(undefined);
+    }, [onChangeSelectedAttachment]);
+
+    const {
+        response,
+    } = useRequest<FileUploadResponse>({
+        skip: isNotDefined(attachmentId) || isDefined(link),
+        url: `server://files/${attachmentId}`,
+    });
```

Comment:
Don't we have a gql node for this?

Reply:
Actually, we don't have the graphgl node for the link upload. so for now it is handled by form.